data/card_catalogue/upgradeable-1.json

```diff
@@ -39,166 +39,6 @@
       "image": "assets/core/card/illustration/Launch.png"
     }
   },
-  {
-    "title": "Flirt",
-    "upgrade": "Charm",
-    "tier": "blue",
-    "details": {
-      "rules": "Increases your rapport with a visible character",
-      "flavour": "If you were words on a card, you'd be some fine print.",
-      "image": "assets/core/card/illustration/Flirt.png"
-    }
-  },
-  {
-    "title": "Charm",
-    "upgrade": "Arouse",
-    "tier": "green",
-    "details": {
-      "rules": "Causes a visible character to like you.",
-      "flavour": "Do you like raisins? How do you feel about a date?",
-      "image": "assets/core/card/illustration/Charm.png"
-    }
-  },
-  {
-    "title": "Arouse",
-    "upgrade": "Seduce",
-    "tier": "red",
-    "details": {
-      "rules": "Causes a visible character to become attracted to you.",
-      "flavour": "There’s only one thing I want to change about you, and that’s your last name.",
-      "image": "assets/core/card/illustration/Arouse.png"
-    }
-  },
-  {
-    "title": "Seduce",
-    "upgrade": "",
-    "tier": "gold",
-    "details": {
-      "rules": "Force any character you can name to act on their desire for you.",
-      "flavour": "Here I am - Your other two wishes?",
-      "image": "assets/core/card/illustration/Seduce.png"
-    }
-  },
-  {
-    "title": "Imitate",
-    "upgrade": "Copy",
-    "tier": "blue",
-    "details": {
-      "rules": "Copy the last Blue card used by any character",
-      "flavour": "This is how stupid you look, watch this.",
-      "image": "assets/core/card/illustration/Imitate.png"
-    }
-  },
-  {
-    "title": "Copy",
-    "upgrade": "Master Mimic",
-    "tier": "green",
-    "details": {
-      "rules": "Copy the last Green card used by any character",
-      "flavour": "Stop copying me. Stop copying me.",
-      "image": "assets/core/card/illustration/Copy.png"
-    }
-  },
-  {
-    "title": "Master Mimic",
-    "upgrade": "The Highest Form of Flattery",
-    "tier": "red",
-    "details": {
-      "rules": "Copy the last Red card used by any character",
-      "flavour": "“Through others we become ourselves.” ― Mad Dog Millson, revered pirate and theif",
-      "image": "assets/core/card/illustration/Master Mimic.png"
-    }
-  },
-  {
-    "title": "The Highest Form of Flattery",
-    "upgrade": "",
-    "tier": "gold",
-    "details": {
-      "rules": "Copy the last Gold card used by any character",
-      "flavour": "Nothing is original.",
-      "image": "assets/core/card/illustration/The Highest Form of Flattery.png"
-    }
-  },
```

Comment:
I deleted this one. I don't think it adds much to the series and has massive disappointment potential in the gold pool if it's the first gold card to ever appear.

Also, a gold card might not've been cast for the last few sessions and we might not even remember what it was